book/forms.rst

```diff
@@ -1021,6 +1021,29 @@ to the ``form()`` or the ``form_start()`` helper:
     a PUT, PATCH or DELETE request. Read the cookbook chapter
     ":doc:`/cookbook/routing/method_parameters`" for more information.
 
+.. note::
+
+  Now that the ``createForm()`` method requires the FQCN, you can not longer pass in ``__construct()`` arguments. To get 
+  around this, you will need to create the default options in the form class ``configureOptions()`` method::
+  
+    use Symfony\Component\OptionsResolver\OptionsResolver;
+    use Symfony\Component\Form\Extension\Core\Type\PasswordType;
+    use Symfony\Component\Form\Extension\Core\Type\TextType;
+
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+      $builder->add('username', TextType::class, array('required' => $options['username_required']));
```

Comment:
In the code samples we use the same rules of Symfony code, so please indent code with 4 white spaces. Thanks!